README.md

```diff
@@ -1,171 +1,26 @@
-1. Complete any required paperwork
-1. Sign up for an "Company" Windows Developer Account https://developer.microsoft.com/en-us/store/register. 
-    > Note: This can take a week or more since you'll be required to confirm your organization's identity with an independent verification service via email and/or telephone.
-
-#### Publishing Code changes
-You'll also need to change a few small things in your project to prepare your distro for publishing to the Windows store
-
-1. In your appxmanifest, you will need to change the values of the Identity field to match your identity in your Windows Store account:
-
-``` xml
-<Identity Name="1234YourCompanyName.YourAppName"
-          Version="1.0.1.0"
-          Publisher="CN=12345678-045C-ABCD-1234-ABCDEF987654"
-          ProcessorArchitecture="x64" />
-```
-
-  > **NOTE**: Visual Studio can update this for you! You can do that by right-clicking on "DistroLauncher-Appx (Universal Windows)" in the solution explorer and clicking on "Store... Associate App with the Store..." and following the wizard. 
-
-2. You will either need to run `build rel` from the command line to generate the Release version of your appx or use Visual Studio directly to upload your package to the store. You can do this by right-clicking on "DistroLauncher-Appx (Universal Windows)" in the solution explorer and clicking on "Store... Create App Packages..." and following the wizard.
-
-Also, make sure to check out the [Notes for uploading to the Store](https://github.com/Microsoft/WSL-DistroLauncher/wiki/Notes-for-uploading-to-the-Store) page on our wiki for more information.
+# This repository
+This repository contains code relevant to Ubuntu on WSL. It builds on top of [Microsoft's reference implementation](https://github.com/microsoft/WSL-DistroLauncher). Head there for any documentation needs.
 
 # Issues & Contact
-Any bugs or problems discovered with the Launcher should be filed in this project's Issues list. The team will be notified and will respond to the reported issue within 3 (US) working days.
-
-You may also reach out to our team alias at wslpartners@microsoft.com for questions related to submitting your app to the Microsoft Store.
-
-# Contributing
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+Any bugs or problems discovered with the Launcher should be filed in this project's Issues list. You may alternatively submit them to [Microsoft's repository](https://github.com/microsoft/WSL-DistroLauncher/issues) if you believe the issue is not particular to Ubuntu. We also have a [page on Launchpad](https://launchpad.net/ubuntuwsl); we check it less often but you may find part of the community there.
```

Comment:
Not many users are expected to be able to know whether a bug is in the launcher or some other component. I'd rephrase this statement to say something like (feel free to improve the text):

```
If you find bugs you believe are particular to Ubuntu running on WSL you are welcome to [create a new issue](https://github.com/ubuntu/WSL/issues/new/choose) in this repository.
Alternatively, if you believe the bug is not specific to Ubuntu, but rather something that should be addressed by the WSL team, head over to [Microsoft's WSL page on GitHub](https://github.com/microsoft/WSL/issues/).
We also have a [page on Launchpad](https://launchpad.net/ubuntuwsl); we check it less often but you may find part of the community there.
You can find news and interesting discussions about Ubuntu on WSL in [Ubuntu's Discourse page](https://discourse.ubuntu.com/c/wsl2/27)
```

Reply:
Also, the majority of bug reports targetting Microsoft are expected in microsoft/WSL instead of the microsoft/WSL-DistroLauncher.